test/e2e/vrfstest/vrf_test.go

```diff
@@ -0,0 +1,145 @@
+package vrfstest
+
+import (
+	"strings"
+	"testing"
+
+	root "github.com/equinix/metal-cli/internal/cli"
+	outputPkg "github.com/equinix/metal-cli/internal/outputs"
+	"github.com/equinix/metal-cli/internal/vrf"
+	"github.com/equinix/metal-cli/test/helper"
+	"github.com/spf13/cobra"
+)
+
+func TestCli_Vrf_Create(t *testing.T) {
+	subCommand := "vrf"
+	consumerToken := ""
+	apiURL := ""
```

Comment:
The `unsupported protocol scheme` error is happening because the test is explicitly setting `apiURL` to an empty string.  There was a bug where the `equinix-sdk-go` setup code didn't actually allow overriding the API URL, but that was change to support mocking certain endpoints in tests.  I believe there's a function in `helper.go` for getting the default (real) API URL for e2e tests.